src/styles/_variables.scss

```diff
@@ -1,14 +1,36 @@
 // Text
-$hacks-h1: 48px;
-$hacks-h2: 36px;
-$hacks-text: 24px;
+$hacks-h1: 3.6em;
+$hacks-h2: 2.4em;
+$hacks-text: 1.8em;
 $hacks-text-spacing: 2em;
+$hacks-small-caps-size: 2.2em;
 $hacks-small-caps-weight: 600;
+$hacks-hamburger-size: 8em;
+$hacks-mobile-menu-text: em(72px);
 
 $hacks-border: 5px dotted rgba(white, 0.5);
+$hacks-three-dee: 3px 3px
+    rgba(
+      $color: red,
+      $alpha: 0.4,
+    ),
+  -3px -3px
+    rgba(
+      $color: cyan,
+      $alpha: 0.4,
+    );
+$hacks-title-three-dee: 2px 0px
+    rgba(
+      $color: red,
+      $alpha: 0.4,
+    ),
+  -2px 0px rgba(
+      $color: cyan,
+      $alpha: 0.4,
+    );
 
 // Sizes
-$hacks-page-width: 981px;
+$hacks-page-width: em(981);
```

Comment:
fancy sass :open_mouth:

Reply:
undid it because it don't work doe

Reply:
you baited me into reading https://adamwathan.me/dont-use-em-for-media-queries/

Reply:
wait where's the undo

Reply:
I'm going to scream brb